src/win32/mouse.c

```diff
@@ -20,13 +20,30 @@
 	: ((button) == RIGHT_BUTTON ? MOUSEEVENTF_RIGHTDOWN \
 	: MOUSEEVENTF_MIDDLEDOWN))
 
+int CalculateAbsoluteCoordinateX(int x) {
+	MMSize displaySize = getMainDisplaySize();
+	return (x * 65536) / displaySize.width;
```

Comment:
**Issue:** Please introduce a constant holding `65536` and consider an explanatory comment linking to the docs. Otherwise it's really hard to guess what this magic number is used for.